### PR TITLE
CTED-1927 add hsuforum updates

### PR DESCRIPTION
**Pull request by loyisongq**

### CODE REVIEW

File: renderer.php [hunk 1]
```diff
@@ -838,9 +833,9 @@ protected function post_walker($cm, $discussion, $posts, $parent, $canreply, &$c
 
                     if ($post->nestedreplycount > 2 && $depth == 0) {
                         // Adding collapsable elements
-                        $output .= "<div id=".$post->id." class='posts-collapse-container collapse'>";
+                        $output .= "<div id=id".$post->id." class='posts-collapse-container collapse'>";
```
Comment (JJSwanevelder) [inline renderer.php:836]: Did this not create an issue with opening up the collapsed posts

**Review by JJSwanevelder — CHANGES_REQUESTED** [review]
See question

File: styles.css [hunk 2]
```diff
@@ -347,6 +347,17 @@ color: white !important;
     color: [[setting:newavatarbadgehovertextcolour]] !important;
 }
 
+.hsuforum-unreadcount {
+    background-color: [[setting:newavatarbadgebackgroundcolour]] !important;
+    color: [[setting:newavatarbadgetextcolour]] !important;
+    border-radius: [[setting:newavatarbadgeborderradius]];
+    border-color: [[setting:newavatarbadgebordercolour]];
```
Comment (JJSwanevelder) [inline styles.css:354]: Was this working on your local. In getsmarter I do not thing we handle settings this way anymore.

Comment (loyisongq) [reply styles.css:354]: yes the button also working on prep310 staging, i just forgot to add for the post new link

File: renderer.php (hunk 1 above)
Comment (loyisongq) [reply renderer.php:836]: without the "id{post->id}" the collapse wouldn't fire

File: styles.css (hunk 2 above)
Comment (JJSwanevelder) [reply styles.css:354]: 👍

File: renderer.php (hunk 1 above)
Comment (JJSwanevelder) [reply renderer.php:836]: Happy

**Review by JJSwanevelder — APPROVED** [review]
👍

**Pull request merged**